docs/REST API/rest-api.md

```diff
@@ -3218,6 +3220,122 @@ A `team_id` of `0` returns the statistics for hosts that are not part of any tea
 }
 ```
 
+### Get host's scripts
+
+_Available in Fleet Premium_
+
+`GET /api/v1/fleet/hosts/{id}/scripts`
+
+#### Parameters
+
+| Name            | Type    | In    | Description                                                                                                                   |
+| --------------- | ------- | ----- | ----------------------------------------------------------------------------------------------------------------------------- |
+| page            | integer | query | Page number of the results to fetch.                                                                                          |
+| per_page        | integer | query | Results per page.                                                                                                             |
+
+#### Example
+
+`GET /api/v1/fleet/hosts/{id}/scripts`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "scripts": [
+    {
+      "script_id": 3,
+      "name": "remove-zoom-artifacts.sh",
+      "last_execution": {
+        "execution_id": "e797d6c6-3aae-11ee-be56-0242ac120002",
+        "executed_at": "2021-12-15T15:23:57Z",
+        "status": "error"
+      }
+    },
+    {
+      "script_id": 5,
+      "name": "set-timezone.sh",
+      "last_execution": {
+        "id": "e797d6c6-3aae-11ee-be56-0242ac120002",
+        "executed_at": "2021-12-15T15:23:57Z",
+        "status": "pending"
+      }
+    },
+    {
+      "script_id": 8,
+      "name": "uninstall-zoom.sh",
+      "last_execution": {
+        "id": "e797d6c6-3aae-11ee-be56-0242ac120002",
+        "executed_at": "2021-12-15T15:23:57Z",
+        "status": "ran"
+      }
+    }
+  ],
+  "meta": {
+    "has_next_results": false,
+    "has_previous_results": false
+  }
+}
+
+```
+
+### Get host's activities
+
+Returns a list of the activities performed in Fleet affecting the current host.
+
+`GET /api/v1/fleet/hosts/{id}/activities`
+
+##### Parameters
+
+| Name  | Type   | In   | Description                        |
+| ----- | ------ | ---- | ---------------------------------- |
+| id | integer | path | **Required**. The ID of the host to get the details for. |
+| page | integer | query | Page number of the results to fetch. |
+| per_page | integer | query | Results per page. |
+| order_key | string | query | What to order results by. Can be any column in the `activites` table. |
```

Comment:
```suggestion
| order_key | string | query | What to order results by. Can be any column in the `activities` table. |
```

Small typo. 

Also, what does "table" mean to the user? I'm guessing we're referring to the Fleet DB table but users don't have access to this. Maybe we list out all the attributes you can sort by?

Heads up, I think we want to solve the same problem for `GET /activities`: https://fleetdm.com/docs/rest-api/rest-api#activities

Reply:
Also, can we cut `order_key` and `order_direction`? It's simpler. The UI doesn't need it.

Reply:
> Also, can we cut order_key and order_direction? It's simpler. The UI doesn't need it.

@noahtalerman I did this to be consistent with what we did for `GET /api/v1/fleet/activities`.

If we cut this, I guess we'll need to define the default order key and direction that we want in the UI. (`created_at` and `desc`), which is fine, we can always improve later if necessary.